.circleci/config/jobs/test-sql.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  image: 'ubuntu-1604:201903-01'
```

Comment:
Not that it necessarily matters but any reason to use such an old machine image? LTS is reasonable but why not 20.04?

Reply:
I picked this to just follow what the [existing jobs was using](https://github.com/hashicorp/boundary/blob/tmessi-wh-test-baseline/.circleci/config/jobs/build.yml) to be consistent.